internal/pod_models.go

```diff
@@ -1007,3 +1015,8 @@ func GetObjectMetadata(cluster *fdbtypes.FoundationDBCluster, base *metav1.Objec
 
 	return *metadata
 }
+
+// GetPodDNSName determines the fully qualified DNS name for a pod.
+func GetPodDNSName(cluster *fdbtypes.FoundationDBCluster, podName string) string {
+	return fmt.Sprintf("%s.%s.%s.svc.%s", podName, cluster.Name, cluster.Namespace, cluster.GetDNSDomain())
```

Comment:
I wonder if we run in some possible issues here for longer Pod or cluster names (https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#names) but I would guess we should be fine.